File: BEE 1046.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args) throws IOException{
        Scanner input = new Scanner(System.in);
        int x = input.nextInt();
        int y = input.nextInt();
        int ans;
        if (x < y){
            ans = y - x;
        }
        else{
            ans = (24 - x) + y;
        }
        System.out.printf("O JOGO DUROU %d HORA(S)\n", ans);
    }
}

File: BEE 1046.php
<?php
    list($starting_time, $ending_time) = explode(" ", readline());
    if($starting_time < $ending_time){
        $ans = $ending_time - $starting_time;
        echo "O JOGO DUROU {$ans} HORA(S)\n";
    }
    else{
        $ans = (24 - $starting_time) + $ending_time;
        echo "O JOGO DUROU {$ans} HORA(S)\n";
    }
?>